StackExchangeRedis/NHibernate.Caches.StackExchangeRedis/RedisCacheProvider.cs

```diff
@@ -28,6 +28,16 @@ public static RedisCacheConfiguration DefaultCacheConfiguration
 			set => _defaultCacheConfiguration = value ?? new RedisCacheConfiguration();
 		}
 
+		/// <summary>
+		/// Set the region configuration.
+		/// </summary>
+		/// <param name="regionName">The region name.</param>
+		/// <param name="configuration">The region configuration.</param>
+		public static void SetRegionConfiguration(string regionName, RegionConfig configuration)
```

Comment:
To allow override of virtual methods please make _connectionMultiplexer protected.

Reply:
Could you "SetRegionConfiguration" method to other cache providers?

Reply:
> please make _connectionMultiplexer protected

It was intentionally made private as the derivered type may use multiple connections (check [DistributedRedisCacheProvider](https://github.com/nhibernate/NHibernate-Caches/blob/790b3841daf12859a2635d82dd96840168fff85b/StackExchangeRedis/NHibernate.Caches.StackExchangeRedis.Tests/Providers/DistributedRedisCacheProvider.cs)), but as such case is rare, I've still made it protected.

> Could you "SetRegionConfiguration" method to other cache providers?

Will do it in a separate PR.